### DISCUSSION TITLE
Can't find IoTWireless anywhere

### DISCUSSION THREAD

**Discussion by tmbs** [aws/aws-sdk-js-v3#2242] [General]
Hi,
I'm sorry but I can't figure out where to find https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/IoTWireless.html in v3, could somebody help me out, please?

**Comment by trivikr** [comment]
The aws-sdk-js-v3 API models hasn't been updated after 2020-12-15, and we're working on fixing issues with smithy models.
IoTWireless was generally available after 2929-12-15, and would be introduced once the models are updated.
